Tests/AppTests/SearchTests.swift

```diff
@@ -21,6 +21,8 @@ import XCTVapor
 
 class SearchTests: AppTestCase {
     
+    let recordSQL = false
```

Comment:
This and the move to using inline snapshots is purely quality of life. I'm updating the query a lot at the moment as we bring more data in for search improvements - updating these tests is a bit of a chore. This lovely lil record boolean makes it a tonne easier.

Reply:
I think

```swift
        SnapshotTesting.isRecording = true
```

should work for `_assertInlineSnapshot` as well. It's what we use for a temporary override with all the other snapshots without needing to add `record: true` to the individual calls (see `SnapshotTestCase.swift` - but it can be added anywhere).